Octokit.Tests.Integration/Clients/RepositoryCommentsClientTests.cs

```diff
@@ -367,4 +367,65 @@ public void Dispose()
             _context.Dispose();
         }
     }
+
+    public class TheCreateReactionMethod : IDisposable
+    {
+        private readonly IGitHubClient _github;
+        private readonly RepositoryContext _context;
+
+        public TheCreateReactionMethod()
+        {
+            _github = Helper.GetAuthenticatedClient();
+
+            _context = _github.CreateRepositoryContext("public-repo").Result;
+        }
+
+        private async Task<Commit> SetupCommitForRepository(IGitHubClient client)
+        {
+            var blob = new NewBlob
+            {
+                Content = "Hello World!",
+                Encoding = EncodingType.Utf8
+            };
+            var blobResult = await client.Git.Blob.Create(_context.RepositoryOwner, _context.RepositoryName, blob);
+
+            var newTree = new NewTree();
+            newTree.Tree.Add(new NewTreeItem
+            {
+                Type = TreeType.Blob,
+                Mode = FileMode.File,
+                Path = "README.md",
+                Sha = blobResult.Sha
+            });
+
+            var treeResult = await client.Git.Tree.Create(_context.RepositoryOwner, _context.RepositoryName, newTree);
+
+            var newCommit = new NewCommit("test-commit", treeResult.Sha);
+
+            return await client.Git.Commit.Create(_context.RepositoryOwner, _context.RepositoryName, newCommit);
+        }
+
+        [IntegrationTest]
+        public async Task CanCreateReaction()
+        {
+            var commit = await SetupCommitForRepository(_github);
+
+            var comment = new NewCommitComment("test");
+
+            var result = await _github.Repository.Comment.Create(_context.RepositoryOwner, _context.RepositoryName,
+                commit.Sha, comment);
+
+            Assert.NotNull(result);
+
+            var newCommitCommentReaction = new NewCommitCommentReaction(Reaction.Confused);
+            var reaction = await _github.Repository.Comment.CreateReaction(_context.RepositoryOwner, _context.RepositoryName, result.Id, newCommitCommentReaction);
+
+            Assert.IsType<CommitCommentReaction>(reaction);
```

Comment:
Maybe a good idea to assert the fields of the response message (eg confirm that the ReactionType you requested is what was actually set).  Also given the other comment about `plus_one` vs `+1` perhaps looping through all of the available `ReactionType` enum values and creating each one woudl be good, just to prove that all of them are defined correctly